### DISCUSSION TITLE
LUX: Visualisation Ka Shishka

### DISCUSSION THREAD

**Discussion by AzamBukhari** [AammarTufail/python_ka_chilla2023#96] [General]
https://buff.ly/3mO5r4g

**Comment by shahbaz-prog** [comment]
Thanks for sharing Azam Bhai.

**Comment by Muhammad-Ali-Butt** [comment]
bohat aala azam bhai btw isko b check karay: [i think ye dataprep ziada behtar ha](https://miro.medium.com/max/720/1*QsgHdfP13pfHCR7jd7-RzQ.webp)  

[Og Article](https://towardsdatascience.com/comparing-five-most-popular-eda-tools-dccdef05aa4c)
